From: Julian Ospald <hasufell@gentoo.org>
Date: Tue Dec  2 20:18:01 UTC 2014
Subject: fix build by adding missing libraries to linker

--- src/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -1083,7 +1083,12 @@
 	${wesnothd_SRC}
 )
 
-target_link_libraries(wesnothd wesnoth-core ${server-external-libs})
+target_link_libraries(wesnothd
+	wesnoth-core
+	${server-external-libs}
+	-lboost_locale
+	-lboost_filesystem
+)
 set_target_properties(wesnothd PROPERTIES OUTPUT_NAME ${BINARY_PREFIX}wesnothd${BINARY_SUFFIX})
 
 install(TARGETS wesnothd DESTINATION ${BINDIR})
@@ -1110,6 +1115,8 @@
 	wesnoth-core
 	${server-external-libs}
 	${SDLNET_LIBRARY} 				# NEEDED with ANA
+	-lboost_locale
+	-lboost_filesystem
 	${LIBINTL_LIBRARIES}
 	)
 set_target_properties(campaignd PROPERTIES OUTPUT_NAME ${BINARY_PREFIX}campaignd${BINARY_SUFFIX})
